index.html

```diff
@@ -23,6 +23,7 @@
     <script src="../scripts/secrets.js"></script>
     <script src="../scripts/connector.js"></script>
     <script src="../scripts/handlers.js"></script>
+    <link rel="stylesheet" href="styles/style.css">
```

Comment:
These are the script tags. Please move the css with the css tags
https://github.com/dev-prakhar/push-me-out-client/blob/main/index.html#L15